.goreleaser.yml

```diff
@@ -118,11 +118,20 @@ snapcrafts:
     grade: stable
     confinement: strict
     publish: true
+    extra_files:
+      - source: config/snap/ttn-lw-stack.wrapper
```

Comment:
Is it a convention in snapcraft ecosystem to add `.wrapper` prefix to these?

Reply:
Yes, and most of the time they are automatically created when using snapcraft directly.